modules/cluster-logging-release-notes-5.3.z.adoc

```diff
@@ -1,7 +1,43 @@
 //Z-stream Release Notes by Version
+[id="cluster-logging-release-notes-5-3-6"]
+= OpenShift Logging 5.3.6
+[role="_abstract"]
+This release includes link:https://access.redhat.com/errata/RHBA-2022:1377[RHBA-2022:1377 OpenShift Logging Bug Fix Release 5.3.6]
+
+[id="openshift-logging-5-3-6-bug-fixes"]
+== Bug fixes
+* Before this update, defining a toleration with no key and the existing operator caused the operator to be unable to complete an upgrade. With this update, this toleration no longer blocks the upgrade from completing. (link:https://issues.redhat.com/browse/LOG-2126[LOG-2126])
+
+* Before this change, it was possible for the collector to generate a warning where the chunk byte limit was exceeding an emitted event. This change allows you to tune the readline limit to resolve the issue as advised by the upstream documentation. (link:https://issues.redhat.com/browse/LOG-2380[LOG-2380])
```

Comment:
```suggestion
* Before this change, it was possible for the collector to generate a warning where the chunk byte limit was exceeding an emitted event. With this change, you can tune the readline limit to resolve the issue as advised by the upstream documentation. (link:https://issues.redhat.com/browse/LOG-2380[LOG-2380])
```
- Per IBM SG, avoid anthropomorphism as much as possible. Focus technical information on users
and the actions that they perform, not on the product and the actions that it performs, like "allowing" or "enabling".